Linkta-org/core
# Incorporates Loader state UI during LLM processing

## Description



- Added `Loader` component with MUI `CircularProgress`.
- LoaderStore added to Zustand Stores.
- Loader added to `UserInputForm`
- Loader added to `OutputVisualizationPage` with zustand state to allow for re-rendering when `regenerate` is clicked.

Fixes #78  

## Type of change




## How Can this be tested? Testing Plan to review this PR



  - `UserInputForm.tsx` and `OutputVisualizationPage.tsx`
  - From the `/generate` route:
    - Enter input and click the `Generate` button to create a new LinktaFlow.
  - From the side nav:
    - Click the `•••` icon and select `Regenerate`.
  - An animated circular loader will appear in the center of the screen until LLM processing is finished.
## Checklist:




## Additional Information:
